packages/helm-charts/testnet/templates/validators.statefulset.yaml

```diff
@@ -155,7 +155,7 @@ spec:
           [[ "$GETH_DEBUG" == "true" ]] && PROXY_ALLOW_PRIVATE_IP_FLAG="--proxy.allowprivateip"
 
           PROXIED_FLAGS=""
-          [ "$RID" -lt "{{ .Values.geth.proxiedValidators }}" ] && PROXIED_FLAGS="--proxy.proxied --proxy.proxyenodeurlpair=`cat /root/.celo/proxyInternalEnode`;`cat /root/.celo/proxyExternalEnode` --nodiscover $PROXY_ALLOW_PRIVATE_IP_FLAG"
+          [ "$RID" -lt "{{ .Values.geth.proxiedValidators }}" ] && PROXIED_FLAGS="--proxy.proxied --proxy.proxyenodeurlpair=`cat /root/.celo/proxyInternalEnode`;`cat /root/.celo/proxyExternalEnode` --nodiscover $PROXY_ALLOW_PRIVATE_IP_FLAG --proxy.allowprivateip"
```

Comment:
The `--proxy.allowprivateip` flag is already present if GETH_DEBUG is true.  Was this failing even with GETH_DEBUG=true in .env?

Reply:
Nice catch.  I just tested with GETH_DEBUG=true and it uses that flag.